icepyx/core/query.py

```diff
@@ -531,6 +136,41 @@ def tracks(self):
         else:
             return sorted(set(self._tracks))
 
+    def _get_concept_id(self, product, version) -> Union[str, None]:
+        collections = earthaccess.search_datasets(
+            short_name=product, version=version, cloud_hosted=True
+        )
+        if collections:
+            return collections[0].concept_id()
+        else:
+            return None
+
+    @property
+    def order_vars(self) -> Union[list[str], None]:
+        """This used to print the list of vasriables for subsetting, Harmony doesn't provide that for IS2 datasets.
+        we do need to implement a class that gets the variables even if it'sm only for listing.
+        """
+        if self.product:
+            return Variables(product=self.product).avail()  # type: ignore[no-any-return]
+        return []
```

Comment:
I'm working on keeping it possible for the user to still see what variables are available for the product, even though they can't subset by them. I'm thinking this is a good place for a deprecation notice that "variable subsetting during order/download" is no longer available, and we should add a new `product_variables` property or something that lets the user just see the list of what would be available before they download. Thoughts?

Reply:
I like `product_variables` or `collection_variables`, agree, renaming the method and removing "order" from the references will make things clearer.